### DISCUSSION TITLE
Manual page views not working as expected

### DISCUSSION THREAD

**Discussion by rimonhanna** [plausible/analytics#1564] [Anything Else]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Describe the bug

window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }
let plausibleUrl = "https://mywebsite.com/" + window.location.href.split(".")[0].split("//")[1] + window.location.href.split(".com")[1];
plausible('pageview', { u:  plausibleUrl });

I want to log subdomains views as custom url page views instead of defining them as new websites, this way I can still have the full site analytics and the subdomains all in one place, I'm doing this by adding the subdomain as if it's part of the path.

so I'm calling the plausible('pageview', { u:  plausibleUrl }); function with https://mywebsite.com/subdomain/page  instead of https://subdomain.mywebsite.com/page

But instead I'm getting only page views for /page without the path

I'm using the plausible.manual.js file

### Expected behavior

Plausible would log the page view as /path/page which is for me the equivalent of /subdomain/page in this case

### Environment

```markdown
- OS: MacOS
- Browser: Edge
- Browser Version: 96
```

**Comment by ukutaht** [comment]
Do you have a link to a page where you have it running? That would make it easier to debug

**Reply by rimonhanna** [reply]
Of course! Here you go: 
https://example-wedding.myknotbook.com/
or
https://example-wedding.myknotbook.com/index

**Reply by ukutaht** [reply]
Looks like you're running self-hosted `v1.4`. The feature you're trying to use (the `u` parameter in manual mode) was added after `v1.4` was released. We release every 6 months or so, the next one will probably be in spring next year

**Reply by rimonhanna** [reply]
Thanks for the explanation, excuse my ignorance, but how in the documentation can I differentiate features that haven't been released to self-hosted yet? So far I was assuming that using the latest released version would include all the features in the docs.

**Reply by metmarkosaric** [reply]
Self-hosted is a long term release so latest features won’t be immediately available as they’re battled-tested in the cloud before released to self-hosted. Every six months we combine all the changes into a new self-hosted release. You can find the latest self-hosted releases and the features (and what's yet to be released to self-hosted) here https://github.com/plausible/analytics/blob/master/CHANGELOG.md

**Reply by rimonhanna** [reply]
Thank you

**Reply by gowthamgts** [reply]
hey @metmarkosaric @ukutaht - is this the same reason why https://github.com/plausible/analytics/discussions/1570 is happening?
